mitx/lms/static/sass/views/_about.scss

```diff
@@ -220,3 +220,39 @@
         }
     }
 }
+
+.student-quote {
+    font-size: 1.4em;
+    width: 100%;
+    margin:50px auto;
+    font-family: Open Sans;
+    font-style: italic;
+    color: #555555;
+    padding: 1.2em 30px 1.2em 75px;
+    border-left: 8px solid $rouge;
+    line-height: 1.6;
+    position: relative;
+    background:#EDEDED;
+
+        &:before {
+        font-family: Arial;
+        content: "\201C";
+        color: $rouge;
+        font-size:4em;
+        position: absolute;
+        left: 10px;
+        top: -10px;
+      }
+
+      &:after {
+        content: '';
+      }
+
+       span {
```

Comment:
If we convert <span> to div <div> we can remove some attributes like   `display: block;` and `margin-top: 1em;`

Reply:
I have tried removing `margin-top`, the gap is not enough.

![Screenshot 2023-06-08 at 2 54 40 PM](https://github.com/mitocw/openlearninglibrary-theme/assets/107186622/3ae8612b-c5c8-4889-8617-2fd7b498f6e6)